nifi-frontend/pom.xml

```diff
@@ -40,6 +40,10 @@
             Name of the theme file to be used.
         -->
         <frontend.theme.name>material</frontend.theme.name>
+        <!--
+            Source directory containing the theme icons. Can be external directory for modular themes.
+        -->
+        <frontend.theme.icons.dir>${frontend.working.dir}/libs/shared/src/assets/themes</frontend.theme.icons.dir>
```

Comment:
shouldn't this be:
```suggestion
        <frontend.theme.icons.dir>${frontend.working.dir}/libs/shared/src/assets/icons</frontend.theme.icons.dir>
```

Reply:
Actually, this is optional so it should be empty.